docs/user/README.md

```diff
@@ -1,44 +1,91 @@
+and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
 
 ![Glean logo](glean.jpeg)
 
-The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+There are two implementations of Glean, with support for 5 different programming languages in total.
+Both implementations strive to contain the same features with similar, but idiomatic APIs.
 
-To contact us you can:
-- Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
-- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
-- Send an email to *glean-team@mozilla.com*.
-- The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
+Unless clearly stated otherwise, regard the text in this book as valid for both clients
+and all the supported programming languages and environments.
+
+### [The Glean SDK](https://github.com/mozilla/glean)
+
+The Glean SDK is an implementation of Glean in Rust, with language bindings for **Kotlin**,
+**Python**, **Rust** and **Swift**.
+
+For development documentation on the `Glean SDK`,
+refer to [the Glean SDK development book](https://mozilla.github.io/glean/).
+
+To report issues or request changes on the Glean SDK,
+file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
+
+### [Glean.js](https://github.com/mozilla/glean.js)
+
+Glean.js is an implementation of Glean in **Javascript**. Currently, it only has support
+for usage in web extensions.
+
+For development documentation on `Glean.js`,
+refer to [the Glean.js development documentation](https://github.com/mozilla/glean.js/tree/main/docs).
 
-The source code is available [on GitHub](https://github.com/mozilla/glean/).
+To report issues or request changes on Glean.js,
+file a bug in [Bugzilla in Data Platform & Tools :: Glean.js][gleanjs-bugs].
 
-## Using this book
+> **Note** Glean.js is still in development and does not provide all the features the Glean SDK does.
+> Feature parity will be worked on after initial validation. Do not hesitate to [file a bug][gleanjs-bugs]
+> if you want to use Glean.js and is missing some key Glean feature.
+## Sections
 
-This book is specifically about **using** the `Glean SDK`.
+### [Using Glean](./user/index.html)
 
-For documentation on **contributing to and developing** the `Glean SDK`, refer to [the Glean SDK development book](../dev/).
+In this chapter we describe how to use Glean in your own libraries and applications.
```

Comment:
```suggestion
In this section we describe how to use Glean in your own libraries and applications.
```

to make it consistent with the next parts. Sorry for suggesting "chapter" to begin with.